web-service-consumer/1.6/modules/ROOT/pages/web-service-consumer-troubleshooting.adoc

```diff
@@ -0,0 +1,166 @@
+    </wsc:connection>
+</wsc:config>
+----
+
+Depending on the server side multipart setting, the server might send some binary data as a Base64-encoded text field or as a referenced binary part in the MIME message.
+
+To troubleshoot how to access the binary data:
+
+* If the server sends the binary data as Base64-encoded text field, you must access this field as part of the message. +
+Base64 decoding is automatically performed by Web Service Consumer.
+* If the server sends the binary data as a referenced binary part, you must access this field as an attachment.
++
+The following example shows an image sent as an attachment with MTOM by a web service server. You use DataWeave to access and transform the image into JSON. In this case, Base64 encoding is required:
++
+[source,DataWeave,linenums]
+----
+import * from dw::core::Strings
+import dw::core::Binaries
+output application/json
+ns nam http://mynamespace.com
+---
+{
+...
+"image": Binaries::toBase64(payload.attachments[0]),
+...
+}
+----
+
+== Troubleshoot Design Time WSDL Issues
+
+Errors related to incorrect WSDL recognition, acquisition, or parsing sometimes lead to the following issues:
+
+* DataSense does not correctly expose the message metadata defined in the WSDL.
+* In the Web Service Consumer Connector configuration screen, the *Operations* field does not show any web service operation names for the connector to invoke:
++
+.Web Service Consumer Connector Operations field
+image::wsc-troubleshoot-1.png[Web Service Consumer Connector Operations field shows empty]
+
+* In the Web Service Consumer Connector global element configuration, the *Port* and *Address* fields are not automatically filled when a WSDL location is provided.
++
+.Web Service Consumer Connector Global Element
+image::wsc-troubleshoot-2.png[In the Web Service Consumer Connector global element window, the port and address fields are not filled when a WSDL location is selected]
+
+To resolve these issues:
+
+* Check the integrity of your WSDL by using your preferred online or desktop viewer applications. +
+Because every parsing error during the configuration is not shown during design-time, implementing an external application helps detect them.
+
+* If you do not add the WSDL as a resource in the application's *src/main/resources* folder in Studio, check for a correct connection to the server hosting the WSDL. +
+Sometimes you need to add custom HTTP settings, such as OAuth settings or user/password settings, to access an HTTP server in an HTTP security layer. See xref:web-service-consumer-configure.adoc[Setting a Custom HTTP Transport Configuration.]
+
+
+[[common-throws]]
+== Understand Common Throws
+
+Here is a list of common throws messages and how to interpret them.
+
+* WSC:SOAP_FAULT
+
+  Error matching the SOAP response with the format provided by the WSDL.
+
+  Every CXF SOAP fault error is wrapped in a WSC:SOAP_FAULT.
```

Comment:
Maybe backtick WSC:SOAP_FAULT? Is that a throw type? It would be good to say what it is.